apps/src/code-studio/pd/foorm/editor/components/FoormEntityLoadButtons.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {Button, DropdownButton, MenuItem} from 'react-bootstrap';
+import {
+  setLastSaved,
+  setSaveError,
+  setHasJSONError,
+  setLastSavedQuestions
+} from '../foormEditorRedux';
+
+class FoormEntityLoadButtons extends React.Component {
+  static propTypes = {
+    resetCodeMirror: PropTypes.func,
+    resetSelectedData: PropTypes.func,
+    onSelect: PropTypes.func,
+    dropdownOptions: PropTypes.array,
```

Comment:
if this is just an array of foom entities, let's call it `entities` or `foormEntities`